pilot/platform/cloudfoundry/config.go

```diff
@@ -35,9 +35,15 @@ type CopilotConfig struct {
 	PollInterval     time.Duration `yaml:"poll_interval" validate:"nonzero"`
 }
 
+// EnvoyConfig describes how the Cloud Foundry platform adapter communicates with Envoy
+type EnvoyConfig struct {
+	Port int `yaml:"port" validate:"nonzero"`
+}
+
 // Config for the Cloud Foundry platform adapter
 type Config struct {
 	Copilot CopilotConfig `yaml:"copilot"`
+	Envoy   EnvoyConfig   `yaml:"envoy"`
```

Comment:
This doesn't sound right. In general, we avoid using proxy from the platform code and vice versa. Everything goes through model. Why do you need this ?

Reply:
since this is supporting an ingress router (gateway basically) we need to define what port it listens on.  we're not entirely clear on where to wire that config from...